test.ps1

```diff
@@ -0,0 +1,3 @@
+Enter-PSHostProcess -Id 40328 -ErrorAction Stop
+                $pid
+                Exit-PSHostProcess
```

Comment:
Please remove.

Reply:
will do. Still getting data. moved to WIP